bin/rucio

```diff
@@ -2420,6 +2420,16 @@ def touch(args):
         client.touch(scope, name, args.rse)
 
 
+# Auto-completers for argument parsing.
+def rse_completer(prefix, parsed_args, **kwargs):
+    """
+    Completes the argument with a list of RSEs
+    """
+    client = get_client(parsed_args)
+    rses = ["%(rse)s" % rse for rse in client.list_rses(None)]
+    return rses
```

Comment:
return [rse for rse in client.list_rses()]
or maybe just 
return client.list_rses() 
I don't remember if list_rses is a generator or not

Reply:
It is a generator, and `argcomplete` didn't seem to like generators.

Cleaned up now -- one still needs the `"%(rse)s" % rse` part because each RSE returned is actually a dictionary (with a key named `rse`), not a string.